source/websocket.c

```diff
@@ -65,9 +65,10 @@ struct aws_websocket {
     struct aws_channel_task move_synced_data_to_thread_task;
     struct aws_channel_task shutdown_channel_task;
     struct aws_channel_task increment_read_window_task;
+    struct aws_channel_task waiting_on_payload_stream_task;
     bool is_server;
 
-    struct {
+    struct thread_data {
```

Comment:
thread_data_t or something, differentiate the name. Old GCC will hate this IIRC

Reply:
I'll just use a comment instead. I just wanted something at the top of this giant block saying what it was